src/pages/study_plans.tsx

```diff
@@ -19,7 +20,7 @@ async function fetcher(url: string): Promise<StudyPlanElement[]> {
 }
 
 export default function StudyPlansPage() {
-	const { data, error, isLoading } = useSWR("https://center-be.stamford.dev/api/study_plans", fetcher);
+	const { data, error, isLoading } = useSWR(`${API_BASE_LINK}/api/study_plans`, fetcher);
```

Comment:
One thing Mr Tawan, I'm gonna stop supporting this endpoint now, can you migrate to the new one?

`/api/resources/study_plans`

https://github.com/stamford-syntax-club/stamfordcenter-backend/issues/15
![image](https://github.com/stamford-syntax-club/stamfordcenter-frontend/assets/92321280/2c57877a-02e8-4441-b07c-f1358a532b04)